recipe/bld.bat

```diff
@@ -19,21 +19,42 @@ if "%ARCH%"=="32" (
     set PLATFORM=x64
 )
 
-
+REM build static library
 msbuild.exe /p:Platform=%PLATFORM% /p:Configuration=Release lib_mpir_gc\lib_mpir_gc.vcxproj
 msbuild.exe /p:Platform=%PLATFORM% /p:Configuration=Release lib_mpir_cxx\lib_mpir_cxx.vcxproj
+REM build dll library
+msbuild.exe /p:Platform=%PLATFORM% /p:Configuration=Release dll_mpir_gc\dll_mpir_gc.vcxproj
```

Comment:
No, in a dll build, it is a little weird, but C++ functions are also in `mpir.dll`